devtools/zenodo/zenodo_data_release.py

```diff
@@ -115,7 +117,6 @@ def get_deposition(self, deposition_id: int) -> _LegacyDeposition:
             method="GET",
             url=f"{self.base_url}/deposit/depositions/{deposition_id}",
             headers=self.auth_headers,
-            timeout=5,
```

Comment:
Removed all of the timeout parameters, since they were just using the default value. This way we can experiment with different defaults / behaviors.